.gitignore/GMRIGHT2_true-coll/Gmright2-coll

```diff
@@ -266,3 +266,88 @@ Dog attack unkwow "code " that's not from here dog location area set in run "tru
 Dog in FTH.dog build source of your string
 Dog.stop struggles
 If the is unlocked leave a trail to confirm parents that  someone else (unkwow.codes) energy enter in here the set the fixing Pall
+# api/config/packages/security.yaml
+security:
+    encoders:
+        App\Entity\User:
+            algorithm: argon2i
+
+    # https://symfony.com/doc/current/security.html#where-do-users-come-from-user-providers
+    providers:
+        # used to reload user from session & other features (e.g. switch_user)
+        app_user_provider:
+            entity:
+                class: App\Entity\User
+                property: email
+
+    firewalls:
+        dev:
+            pattern: ^/_(profiler|wdt)
+            security: false
+        main:
+            stateless: true
+            anonymous: true
+            provider: app_user_provider
+            json_login:
+                check_path: /authentication_token
+                username_path: email
+                password_path: password
+                success_handler: lexik_jwt_authentication.handler.authentication_success
+                failure_handler: lexik_jwt_authentication.handler.authentication_failure
+            guard:
+                authenticators:
+                    - lexik_jwt_authentication.jwt_token_authenticator
+
+    access_control:
+        - { path: ^/docs, roles: IS_AUTHENTICATED_ANONYMOUSLY } # Allows accessing the Swagger UI
+        - { path: ^/authentication_token, roles: IS_AUTHENTICATED_ANONYMOUSLY }
+        - { path: ^/, roles: IS_AUTHENTICATED_FULLY }
+
+# api/config/routes.yaml
+authentication_token:
+    path: /authentication_token
+    methods: ['POST']
+
+_var gmright2 step in separate between both side of world this one set here and the other one set there if this and that met than the computer gmright2 technology is stable
```

Comment:
[The new world.docx](https://github.com/Gmright2-zone/GMRIGHT2-coll/files/5323418/The.new.world.docx)